### ISSUE TITLE
Dark theme

### ISSUE THREAD

**Issue by electro-logic** [chrisg32/AffinityDesignerUWPAssets#1]
Hello,
Thank you for the UWP assets, do you have any update for the dark theme?
Cheers

**Comment by chrisg32** [comment]
@electro-logic No updates and I don’t currently have any plans to add the dark theme. I am not working on any UWP projects at the moment so this is probably a long ways off if at all. Eventually I would like to update to WinUI 3. In the meantime I would welcome any pull requests.

**Comment by electro-logic** [comment]
Fair enough, I was asking just in case was already done. Thanks for the reply.
